docs/content/getting-started/project-file-reference.mdx

```diff
@@ -0,0 +1,458 @@
+---
+title: Dagster project file reference | Dagster Docs
+description: ""
+---
+
+# Dagster project file reference
+
+Following [our recommended best practices for project structure](/guides/dagster/recommended-project-structure), a small Dagster project might look like this:
```

Comment:
We have the best practices for project structure doc, i worry that duplicating it here might get confused with you only need these files and the point of this doc is to say when you need different stuff